containers/doc/server-kubernetes/rhn-ssl-tool.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    run: rhn-ssl-tool
+  name: rhn-ssl-tool
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      initContainers:
+      - name: gen-ca
+        image: registry.opensuse.org/systemsmanagement/uyuni/master/servercontainer/containers/uyuni/server
```

Comment:
is this going to remain the final project used or it is just for testing this first  round of changes?

Reply:
It's just the first one... would have to be changed as soon as we merge in master. I may even drop that file completely